resources/measures/ResidentialGeometryCreateSingleFamilyAttached/measure.rb

```diff
@@ -258,6 +258,9 @@ def run(model, runner, user_arguments)
     foundation_type = runner.getStringArgumentValue('foundation_type', user_arguments)
     foundation_height = runner.getDoubleArgumentValue('foundation_height', user_arguments)
     attic_type = runner.getStringArgumentValue('attic_type', user_arguments)
+    if attic_type == 'finished attic'
+      num_floors -= 1
+    end
```

Comment:
Since we're redefining what the input means, it seems like the num_floors argument description should be updated to clarify.